### DISCUSSION TITLE
Tabs text color

### DISCUSSION THREAD

**Discussion by i-am-zynn** [fyne-io/fyne#3320] [Q&A] [answered]
Hello,

By default the color of the text of the [tabs](https://developer.fyne.io/container/apptabs) is white. I would like to know how to change the color of this text.
I am new to Fyne, I have not found at all.

Thank you in advance.

**Comment by andydotxyz** [comment, marked as answer]
The tabs are a standard component and match the foreground text colour of the theme, so don't support separate colour specification.

As our API is based on intents (like a high importance button or disabled entry) what is the use-case of a different looking tab?
